modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -51,80 +51,181 @@
 /**
   @defgroup calib3d Camera Calibration and 3D Reconstruction
 
-The functions in this section use a so-called pinhole camera model. In this model, a scene view is
-formed by projecting 3D points into the image plane using a perspective transformation.
+The functions in this module use a so-called pinhole camera model. The view of a scene
```

Comment:
> in this module use a so-called pinhole camera model

`fisheye` camera model is in this "module" too (in terms of OpenCV modules). Can we keep "section" here?

Reply:
+1